2018-05-Berlin/attendee-rsvp.md

```diff
@@ -0,0 +1,81 @@
+# Node.js Collaborator Summit, 2018-05-31 - 2018-06-01, Berlin, Germany
+This document serves as an RSVP list for attendees in person, attendees remote, and the collaborator's dinner hosted by the Node Foundation.
+
+This is how we will track amount of food and seats. Submit a PR if you need to RSVP or update your RSVP.
+
+### Attendees In-Person 
+If you will be in Berlin to attend the Collaborator's Summit - add your name.
+
+1. [Myles Borins](https://github.com/MylesBorins) // Organizer
+2. [Matteo Collina](https://github.com/mcollina) // Organizer
+3. [Dan Shaw](https://github.com/dshaw) // Organizer
+4. [olivia](https://github.com/oe) // Organizer
+5. [Tracy Hinds](https://github.com/hackygolucky) // Organizer
+6. [Mark Hinkle](https://github.com/mrhinkle) // Node Foundation
+7. [Tierney Cyren](https://github.com/bnb)
+8. [Anatoli Papirovski](https://github.com/apapirovski)
+9. [Anna Henningsen](https://github.com/addaleax)
+10. [Benedikt Meurer](https://github.com/bmeurer)
+11. [Zeke Sikelianos](https://github.com/zeke)
+12. [Dhruv Jain](https://github.com/maddhruv)
+13. [Ben Michel](https://github.com/obensource)
+14. [Lin Clark](https://github.com/linclark)
+15. [Hassan Sani](https://github.com/inidaname)
+16. [Yunong Xiao](https://github.com/yunong)
+17. [Adam Miller](https://github.com/amiller-gh)
+18. [Daniel Ehrenberg](https://github.com/littledan)
+19. [Ali Ijaz Sheikh](https://github.com/ofrobots)
+20. [Danese Cooper](https://github.com/Danese)
+21. [Thomas Watson](https://github.com/watson)
+22. [Stephen Belanger](https://github.com/Qard)
+23. [Avraam Mavridis](https://github.com/AvraamMavridis)
+24. [Addison Berry](https://github.com/add1sun)
+25. [Yang Guo](https://github.com/hashseed)
+26. [Daniel Bevenius](https://github.com/danbev)
+27. [Abraham Agiri Jr.](https://github.com/codeekage)
+28. [Gibson Fahnestock](https://github.com/gibfahn)
+29. [Vladimir de Turckheim](https://github.com/vdeturckheim)
+30. [Nicola Del Gobbo](https://github.com/NickNaso)
+31. [Shelley Vohr](https://github.com/codebytere)
+32. [Yihong Wang](https://github.com/yhwang)
+33. [Bryan English](https://github.com/bengl)
+34. [Benjamin Gruenbaum](https://github.com/benjamingr)
+35. [Waleed Ashraf](https://github.com/WaleedAshraf)
+36. [James M Snell](https://github.com/jasnell) - MAYBE
+37. [Gus Caplan](https://github.com/devsnek)- MAYBE
+38. [Matheus Marchini](https://github.com/mmarchini)- MAYBE
```

Comment:
I won't be able to attend in person, but I'll participate remotely if that's an option :)